modules/openshift-developer-cli-installing-odo-on-windows.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/installing-odo.adoc     
+
+[id="installing-odo-on-windows"]
+
+= Installing {odo-title} on Windows
+
+== Binary installation
+
+. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-windows-amd64.exe[odo.exe] file.
+. Extract the file and add the location of extracted binary to your `GOPATH/bin` directory.
+
+[discrete]
+== Setting the `PATH` variable for Windows 7/8
```

Comment:
odo is a command-line tool. To run it users will either need to run .exe file manually (aka type in their power shell path to the .exe file every time they want to do smth with odo) or add it to their PATH (and then they can run it typing `odo` command). 

I will need to reword it though since the link doesn't lead to an archive but to an .exe file, there's nothing to unpack there. My mistake.

Reply:
Updated the step.